lib/zold/http.rb

```diff
@@ -129,6 +116,26 @@ def headers
       end
     end
 
+    def build_session(timeout:, additional_header: {})
+      Patron::Session.new(
+        timeout: timeout,
+        connect_timeout: CONNECT_TIMEOUT,
+        base_url: base_url,
+        headers: headers.merge(additional_header)
+      )
+    end
+
+    def base_url
+      "#{@uri.scheme}://#{@uri.host}:#{@uri.port}"
+    end
+
+    def path
+      path = @uri.path
+      path += '?' + @uri.query if @uri.query
+
```

Comment:
@dapi we don't like this, check: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
Ok. You need to describe it with rubocop rules.

Reply:
What do you think about this solution?

`[@uri.path, @ury.query].compact.join(‘?’)`